tests/NexusMods.DataModel.SchemaVersions.Tests/Schema.verified.md

```diff
@@ -17,6 +17,7 @@ models in the app, then validate the tests to update this file.
 | NexusMods.ArchivedFile/NxFileEntry                                                 | Blob                    | False   | False | False     | 
 | NexusMods.Collections.DirectDownloadLibraryFile/LogicalFileName                    | Utf8                    | False   | False | False     | 
 | NexusMods.Collections.DirectDownloadLibraryFile/Md5                                | UInt128                 | True    | False | False     | 
+| NexusMods.Collections.NexusCollectionBundledLoadoutGroup/BundleDownload            | Reference               | False   | False | False     | 
```

Comment:
This won't cause any issues?

Reply:
There is one "issue": collections with bundled mods that have previously been installed will not show up as installed after this PR. The fix is to just install the collection again which will correctly install the bundled mod.

Reply:
Strange I guess, wasn't the case for my existing installed collection

Reply:
This will only be the case if any of those collections had bundled mods.

Reply:
Ah, makes sense